invadedhouse/commands.py

```diff
@@ -125,6 +126,20 @@ def interpret_command(self, cmd: str):
             print('OMG YOU FOUND MY BRILLIANTLY WONDERFUL SECRET!!')
             warp_room = self.house.find_room_by_name('warp room')
             self.player.set_location(warp_room)
+        elif cmd.startswith('loot '):
+            item_str = cmd.replace('loot ','')
+            if self.player.get_location().can_player_pickup(item_str):
+                loot_items = self.player.get_location().get_item(item_str).loot()
+                for item in loot_items:
+                    self.player.pickup(item)
+                print(f'You looted items from {item_str}!')
```

Comment:
You should add the check if the chest is empty and say something different in that case.